readme.md

```diff
@@ -16,22 +16,25 @@ The reason for OpenConcept's efficiency is the analytic derivatives built into e
 
 ## Documentation
 
-Automatically-generated documentation is available at (https://openconcept.readthedocs.io)
+Automatically-generated documentation is available at (https://mdolab-openconcept.readthedocs-hosted.com/en/latest/).
+
+To build the docs locally, install the `sphinx_mdolab_theme` and `redbaron` via `pip`. Then enter the `docs` folder in the root directory and run `make html`. The built documentation can be viewed by opening `_build/html/index.html`.
 
 ## Getting Started
 
-1. Clone the repo to disk
+1. Clone the repo to disk (`git clone https://github.com/mdolab/openconcept`)
```

Comment:
I generally prefer not duplicating these things, so IMO these can be removed and you can just refer people to the docs site.

Reply:
I'd personally like to keep them in the readme so that if someone comes to the repo and just wants to download and install it right away the info is there. I've used it before in other repos.

Reply:
As long as you're willing to maintain both and make sure they're consistent